www.foia.gov/request/index.html

```diff
@@ -1,10 +1,10 @@
 ---
 title: Create a request
-layout: sidebar
+layout: default
 navigation: request
 use-uswds: true
 bundle: request
-grid: usa-grid-full
+grid: usa-grid-full grid-left
```

Comment:
Mentioned this in slack, this is going to affect all the `/request/*` pages, so it includes the request landing page with the agency finder. We probably want to specify no grid here, and then add grid classes in the individual `*Page` components.